spanner/tests/unit/test_streamed.py

```diff
@@ -25,13 +25,16 @@ def _getTargetClass(self):
 
         return StreamedResultSet
 
-    def _make_one(self, *args, **kwargs):
-        return self._getTargetClass()(*args, **kwargs)
+    def _make_one(self, response_iterator, restart=object(), source=None):
+        return self._getTargetClass()(
+            response_iterator, restart, source=source)
 
     def test_ctor_defaults(self):
         iterator = _MockCancellableIterator()
-        streamed = self._make_one(iterator)
+        restart = object()
```

Comment:
`mock.sentinel.restart` may convey slightly more meaning here (not sure how familiar you are with the features of `unittest.mock`)

Reply:
I'll pass here:  identity is the only point.